number_theory/modular.h

```diff
@@ -40,10 +43,13 @@ struct ModulusWrapper {
 
 }  // namespace
 
+// Ring of integers modulo |mod|.
 template <ModulusWrapper mod>
 class Modular {
  public:
+  // The base type of values in the ring.
```

Comment:
I am not sure since we are not English native speakers but I feel like *the values* might be better.

Furthermore, *base* makes this sentence a little bit harder to understand. Isn't it just the type of the values in the ring? Why do we say it is base type? Do we have other types where marking this one as the base type clarifies some details?

Reply:
I think the type of the values in the ring is the Modular class itself, and the base type is the type that the Modular class is based on.